### DISCUSSION TITLE
please help to understand sync

### DISCUSSION THREAD

**Discussion by KvizadSaderah** [TriliumNext/Trilium#2097] [Q&A]

**Comment by maniackcrudelis** [comment]
Hi,

your last edit removed all your question, there's nothing anymore to read...

Anyway, I've spotted that you were speaking about YunoHost and I'm on the same user case.

First, did you install the app as a public app ? The install script clearly states that you can't synchronize your app if the app is private.
And considering that you've got a 302, it looks a lot like a SSO blocked request.

Otherwise, you may have a look to that issue about synchronization, https://github.com/YunoHost-Apps/trilium_ynh/issues/4
